<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="testsuite" duration-ms="99" started-at="2018-08-24T12:19:33Z" finished-at="2018-08-24T12:19:33Z">
    <groups>
    </groups>
    <test name="testname" duration-ms="99" started-at="2018-08-24T12:19:33Z" finished-at="2018-08-24T12:19:33Z">
      <class name="SimpleTest.Test1">
        <test-method status="PASS" signature="setup()[pri:0, instance:SimpleTest.Test1@1aeed2f]" name="setup" is-config="true" duration-ms="416" started-at="2018-08-24T17:49:32Z" finished-at="2018-08-24T17:49:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="Test1()[pri:0, instance:SimpleTest.Test1@1aeed2f]" name="Test1" duration-ms="18" started-at="2018-08-24T17:49:33Z" finished-at="2018-08-24T17:49:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test1 -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:SimpleTest.Test1@1aeed2f]" name="getResult" is-config="true" duration-ms="19" started-at="2018-08-24T17:49:33Z" finished-at="2018-08-24T17:49:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Test1 status=SUCCESS method=Test1.Test1()[pri:0, instance:SimpleTest.Test1@1aeed2f] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="FAIL" signature="Test2()[pri:0, instance:SimpleTest.Test1@1aeed2f]" name="Test2" duration-ms="3" started-at="2018-08-24T17:49:33Z" finished-at="2018-08-24T17:49:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at SimpleTest.Test1.Test2(Test1.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test2 -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:SimpleTest.Test1@1aeed2f]" name="getResult" is-config="true" duration-ms="18" started-at="2018-08-24T17:49:33Z" finished-at="2018-08-24T17:49:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Test2 status=FAILURE method=Test1.Test2()[pri:0, instance:SimpleTest.Test1@1aeed2f] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:SimpleTest.Test1@1aeed2f]" name="teardown" is-config="true" duration-ms="652" started-at="2018-08-24T17:49:33Z" finished-at="2018-08-24T17:49:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- SimpleTest.Test1 -->
    </test> <!-- testname -->
  </suite> <!-- testsuite -->
</testng-results>
